modules/vaos/app/services/eps/appointment_service.rb

```diff
@@ -142,5 +146,39 @@ def build_submit_payload(params)
     def provider_services
       @provider_services ||= Eps::ProviderService.new(user)
     end
+
+    ##
+    # Raises a VAOS::Exceptions::BackendServiceException for EPS error responses
+    #
+    # @param error_message [String] The error message from the EPS response
+    # @param response [Object] The HTTP response object
+    # @raise [VAOS::Exceptions::BackendServiceException]
+    def raise_eps_error(error_message, response)
+      # Log the error without PII - only include safe context information
+      Rails.logger.warn("EPS appointment error detected", {
+        error_type: error_message,
+        method: caller_locations(1, 1)[0].label,
+        status: response.status || 'unknown'
+      })
+
+      # Create a sanitized error body that doesn't contain PII
+      # Only include the error field and minimal context for debugging
+      sanitized_body = {
+        error: error_message,
```

Comment:
This error hash is also different than others we have used. I have seen title, detail and maybe code.